### ISSUE TITLE
About multi-layer forward pass

### ISSUE THREAD

**Issue by daleigehhh** [KimUyen/ConvLSTM-Pytorch#4]
Hi, @KimUyen 
I am working on a local map prediction problem and I want to use Bi-ConvLSTM to solve my problem.
In your implementation ConvLSTM part, line 238:
![ConvLSTM_stacked_forwardpass](https://github.com/user-attachments/assets/b21fdbc6-9eff-4b83-a795-09472fb1506d)
` ## LSTM forward direction
        input_fw = input_tensor
        for layer_num in range(self.layer_num):
            h, c = hidden_states[layer_num]
            output_inner = []
            for t in range(seq_len):
                h, c = self.cells_fw[layer_num](input_tensor=input_fw[:, t, :, :, :], cur_state=[h, c])
                output_inner.append(h)
            layer_output = torch.stack(output_inner, dim=1)
            layer_outputs_fw.append(layer_output)
        layer_outputs = torch.stack(layer_outputs_fw, dim=1)
        last_state_fw = [h, c]`
why the input of every layer is the same, I thought it should be the hidden state of the last layer, am I right?
